src/test/java/com/marizueva/laboratory/hw4/voidpageobject/tests/SiteTestWithHardAssertions.java

```diff
@@ -0,0 +1,104 @@
+package com.marizueva.laboratory.hw4.voidpageobject.tests;
+
+import com.marizueva.laboratory.hw4.testdata.UsedInTestTerms;
+import com.marizueva.laboratory.hw4.utils.testnaming.FeaturesNaming;
+import com.marizueva.laboratory.hw4.utils.testnaming.StoriesNaming;
+import com.marizueva.laboratory.hw4.voidpageobject.pages.HeaderItems;
+import com.marizueva.laboratory.hw4.voidpageobject.pages.HomePage;
+import io.qameta.allure.Feature;
+import io.qameta.allure.Story;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class SiteTestWithHardAssertions extends BaseTest {
```

Comment:
Please check src/test/java/com/marizueva/laboratory/hw4/voidpageobject/tests/FailedSiteTestWithHardAssertions.java for the related comments

Reply:
fixed